inc/template-tags.php

```diff
@@ -99,13 +99,7 @@ function _s_categorized_blog() {
 		set_transient( '_s_categories', $all_the_cool_cats );
 	}
 
-	if ( $all_the_cool_cats > 1 ) {
-		// This blog has more than 1 category so _s_categorized_blog should return true.
-		return true;
-	} else {
-		// This blog has only 1 category so _s_categorized_blog should return false.
-		return false;
-	}
+	return ( $all_the_cool_cats > 1 );
```

Comment:
Maybe summarize the documentation which is now being removed and add it above the line ?